src/page_components/listing/ListingDetailHeader.scss

```diff
@@ -71,23 +90,9 @@
 }
 
 .detail-header__subtitle {
-  @apply text-sm;
+  font-size: var(--bloom-font-size-sm);
```

Comment:
Detroit doesn't yet have the updated names for CSS variables that we did in core. Because of this, this change will actually make the subtitle text smaller than it is now and below an accessible threshold. On uptake, we'll have to do similar work when we renamed a few of the variables in core.